nbproject/private/private.properties

```diff
@@ -0,0 +1,8 @@
+browser.id=SL[/Browsers/FirefoxBrowser
+copy.src.files=false
```

Comment:
please add the nbproject folder to your gitignore

Reply:
ok